bocas/artifacts/metrics.py

```diff
@@ -1,10 +1,22 @@
 from bocas.artifacts.artifact import Artifact
+from .yaml_utils import parse_yaml_node
```

Comment:
can we do `bocas.yaml_utils`

Reply:
(to explain why, it makes it easier to reorganize files down the line)

Reply:
Yep!  Moved it to bocas.yaml_utils.